zenoh-flow/src/runtime/message.rs

```diff
@@ -46,11 +47,22 @@ impl DataMessage {
     }
 }
 
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct RecordingMetadata {
+    pub(crate) timestamp: Timestamp,
+    pub(crate) port_id: PortId,
+    pub(crate) node_id: NodeId,
+    pub(crate) flow_id: FlowId,
+    pub(crate) instance_id: Uuid,
+}
+
 #[derive(Clone, Debug, Serialize, Deserialize)]
 pub enum ControlMessage {
     ReadyToMigrate,
     ChangeMode(u8, u128),
     Watermark,
```

Comment:
As discussed, we can comment them out and indicate that they are not yet defined